Packs/CommonScripts/ReleaseNotes/1_12_55.md

```diff
@@ -0,0 +1,6 @@
+
+#### Scripts
+
+##### New: ReadQRCode
+
+- New: Extracts the text from a QR code. (Available from Cortex XSOAR 6.10.0).
```

Comment:
check that this double New language is standard

Reply:
This is how it was generated